src/api.ts

```diff
@@ -70,30 +70,57 @@ export class GrafanaApiClient implements GrafanaApi {
   }
 
   async dashboardsByTag(tag: string): Promise<Dashboard[]> {
```

Comment:
Probably would call it `tags` as there could be many.

Reply:
I have rebased the changes from main and re-done the changes. I used now `tags` as it makes more sense. Good catch!